package.json

```diff
@@ -57,7 +58,7 @@
     "lint": "eslint 'src/**/*.ts' 'rpc/**/*.ts' ",
     "lint-junit": "eslint 'src/**/*.ts' 'rpc/**/*.ts' --format junit -o reports/junit/lint-junit.xml ",
     "lint-html": "eslint  'src/**/*.ts' 'rpc/**/*.ts' --format html  -o reports/lint.html ",
-    "test": "jest --detectOpenHandles --forceExit --runInBand",
+    "test": "jest --detectOpenHandles --forceExit --runInBand --verbose=false",
```

Comment:
What is the reason for this change? If necessary for this PR, can you please make a note about it in the PR description

Reply:
oh, this is for printing the log when running the tests, forgot to remove it...